docs/Contributing.asciidoc

```diff
@@ -238,12 +238,23 @@ We use annotations in some places to mark "uncoverable" code such as this:
 
 See the docs for details https://metacpan.org/pod/Devel::Cover
 
-To run perltidy tests run:
+There are some ways to save some time when executing local tests:
+
+* One option is selecting individual tests to run as explained above
+* Set the make variable `KEEP_DB=1` to keep the test database process spawned
+  for tests for faster re-runs or run tests with `prove` manually after the
+  test database has been created.
+* Run `tools/tidy --only-changed` to tidy up code before commiting in git
+* Set the environment variable `DIE_ON_FAIL=1` from `Test::Most` for faster
+  aborts from failed tests.
+
+
```

Comment:
Since you need to resolve conflicts anyways you can get rid of the additional blank line.

Reply:
hm, I see. In markdown one would properly end a list with a double blank line. Not necessary here :) Fixed.